solr/solrj/src/test/org/apache/solr/client/solrj/TestSolrJErrorHandling.java

```diff
@@ -102,16 +102,19 @@ public void showExceptions() throws Exception {
 
   @Test
   public void testWithXml() throws Exception {
-    HttpSolrClient client = (HttpSolrClient) getSolrClient();
-    client.setRequestWriter(new RequestWriter());
+    client =
+        new HttpSolrClient.Builder(getServerUrl()).withRequestWriter(new RequestWriter()).build();
+
```

Comment:
`try-with-resources` since we create new clients here.

Reply:
Done